apis/duck/cached_test.go

```diff
@@ -19,24 +19,25 @@ package duck
 import (
 	"context"
 	"fmt"
-	"sync/atomic"
 	"testing"
 	"time"
 
+	"go.uber.org/atomic"
 	"golang.org/x/sync/errgroup"
+
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/client-go/tools/cache"
 )
 
 type BlockingInformerFactory struct {
 	block  chan struct{}
-	nCalls int32
+	nCalls *atomic.Int32
```

Comment:
Pretty much a nit: think you can just use `atomic.Int32` here since the zero value is valid, and then you don't need the NewInt32 later (eg like in [throttler](https://github.com/knative/serving/blob/98f3589c694b7ed987b075c4fb4465edead6e4e9/pkg/activator/net/throttler.go#L80))

Reply:
Done.